cypress/integration/add-delete-collection.ts

```diff
@@ -0,0 +1,36 @@
+before(() => {
+  cy.clearCookies()
+  cy.clearLocalStorage()
+})
+
+it('can add cards to the collection', () => {
+  cy.visit('/')
+  cy.findAllByRole('link', { name: 'Search' }).click()
+  const cardFigures = cy.findAllByRole('figure').should('exist')
+
+  cardFigures.each((cardFigure) => {
+    cy.wrap(cardFigure).realHover()
+    cy.wrap(cardFigure)
+      .findByRole('button')
+      .should('be.visible')
+      //? questionable click workaround
+      .realClick()
```

Comment:
this definitely seems like I hack haha, I'd either use cypress-real-events for your events or the default cypress events